install/macos.md

```diff
@@ -7,16 +7,27 @@ parent: Download
 ---
 # Install on macOS
 
-The recommended installation method is via [Homebrew Cask](https://brew.sh/):
+The recommended installation method is via [Homebrew Cask](https://brew.sh/).
+
+If you plan to use Borg's mount feature to browse backups, you will need to install macFuse and Borg's [own tap](https://github.com/borgbackup/homebrew-tap):
```

Comment:
This is subjective, but current [osxfuse](https://formulae.brew.sh/cask/osxfuse) cask is at version 3.11.2 of "FUSE for macOS" (the project was renamed "macFUSE" starting at version 4. That being said, the upstream naming convention is all over the place... osxfuse vs FUSE for macOS vs macFUSE. Using "macFUSE" is perhaps the future-proof way to go.

Reply:
Right. I got the capitalization wrong here. Wrote "macFUSE" below.